src/applications/personalization/profile/components/Profile.jsx

```diff
@@ -275,10 +276,12 @@ const mapStateToProps = state => {
   const isCNPDirectDepositBlocked = cnpDirectDepositIsBlocked(state);
   const isEligibleToSetUpCNP = cnpDirectDepositAddressIsSetUp(state);
   const is2faEnabled = isMultifactorEnabled(state);
+  const signInService = signInServiceNameSelector(state);
 
   const shouldFetchCNPDirectDepositInformation =
-    isEvssAvailable && is2faEnabled;
-  const shouldFetchEDUDirectDepositInformation = is2faEnabled;
+    isEvssAvailable && signInService === 'idme' && is2faEnabled;
```

Comment:
this simple `signInService === 'idme'` check won't scale when we start to grant other sign in providers access to direct deposit, but this is fine for now.